pages.it/linux/ip.md

```diff
@@ -0,0 +1,36 @@
+# ip
+
+> Mostra / manipola il routing, i dispositivi, i criteri di routing e i tunnel.
+> Alcuni sottocomandi, come `ip address`, hanno una propria documentazione d'uso. Maggiori informazioni: <https://www.man7.org/linux/man-pages/man8/ip.8.html>
+
+- Lista delle interfacce con informazioni dettagliate:
+
+`ip address`
+
+- Lista delle interfacce con informazioni brevi sul livello di rete:
```

Comment:
```suggestion
- Lista le interfacce con informazioni brevi sul livello di rete:
```

Reply:
I changed "Lista" to "Elenca", which means "list" from the verb "to list".